Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1183,6 +1183,13 @@ working, take a look at `this KEP
     free mode, make sure that default Kubernetes services like ``kube-dns`` and ``kubernetes``
     have the required label value.
 
+External access to ClusterIP services
```

Comment:
Nit: Please use title case for consistency with the other section titles.

Reply:
title case'd